### DISCUSSION TITLE
Copilot Plugin

### DISCUSSION THREAD

**Discussion by shanjitsingh1** [apache/netbeans#5392] [Ideas]
Please a copilot plugin for NetBeans please

**Comment by diegofalcinelli** [comment]
YES....

**Comment by Chris2011** [comment]
Wrong Repo. This is about the core not about any 3rd party plugins.

**Reply by BenjaminJamesNexia** [reply]
where to raise this request then?

**Reply by Chris2011** [reply]
At the copilot repo

**Comment by manojkumarredbus** [comment]
Ollama based plugin for Copilot like use https://github.com/manojkumarredbus/netbeans-Ollama

**Comment by jGauravGupta** [comment]
Jeddict AI Assistant : https://plugins.netbeans.apache.org/catalogue/?id=103

<img src="https://github.com/user-attachments/assets/8e3cfb6b-afbe-4115-b7da-9a8976f930ca" width="400"/>
